code/modules/spells/spell.dm

```diff
@@ -14,11 +14,11 @@
 	var/action_icon_state = "spell_default"
 	var/action_background_icon_state = "bg_spell"
 	var/base_action = /datum/action/spell_action
-	var/mob/living/owner
+	var/datum/weakref/owner
 
-/obj/effect/proc_holder/Initialize(mob/living/owner)
+/obj/effect/proc_holder/Initialize(mapload, mob/living/new_owner)
 	. = ..()
-	src.owner = owner
+	owner = WEAKREF(new_owner)
```

Comment:
So nothing else uses this proc_holder `owner` var?

Reply:
I am pretty sure it is just the vat beast. 

Floyd added it to be able to relay abilities to the rider.